src/baseline_bots/utils.py

```diff
@@ -97,44 +106,107 @@ def FCT(string) -> str:
     return f"FCT ({string})"
 
 
+def HUH(string) -> str:
+    """Forms HUH message"""
+    return f"HUH ({string})"
+
+
 def parse_FCT(msg) -> str:
     """Detaches FCT from main arrangement"""
     if "FCT" not in msg:
         raise ParseError("This is not an FCT message")
     try:
-        return msg[5:-1]
+        return msg[msg.find("(") + 1:-1]
     except Exception:
-        raise ParseError(f"Cant parse FCT msg {msg}")
+        raise Exception(f"Cant parse FCT msg {msg}")
 
 
 def parse_PRP(msg) -> str:
     """Detaches PRP from main arrangement"""
     if "PRP" not in msg:
         raise ParseError("This is not an PRP message")
     try:
-        return msg[5:-1]
+        return msg[msg.find("(") + 1:-1]
     except Exception:
-        raise ParseError(f"Cant parse PRP msg {msg}")
```

Comment:
Why make this change?

Reply:
I think this is from merging ALLAN-DIP/main. The only thing I changed in baseline_bots is removing import REJ